doc/source/dev/core-dev/distributing.rst.inc

```diff
@@ -189,6 +151,7 @@ would need to be distributed via custom channels, e.g. in a
 Wheelhouse_, see at the wheel_ and Wheelhouse_ docs.
 
 
+.. _`SciPy's configuration file`: https://github.com/scipy/scipy/blob/main/pyproject.toml
 .. _Numpy: https://numpy.org
```

Comment:
ah, I need to change this for the link, 1 sec

Reply:
The link worked though?

Reply:
right, I suppose this syntax is case-insensitive in rst then. The difference is `Numpy` and `NumPy`, but I guess we can leave it as is.

Reply:
Oops I didn't check all links, just the ones you changed.

Reply:
another reason to get rid of `rst` 😆